### PR TITLE
add vertical-pod-autoscaler images

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related: 

### Pre-review Checklist

- [ ] **IMPORTANT: 'image-request' tag has been applied if this PR is adding any images, including new versions or variants**

#### For new image PRs only

If you have an apko.yaml file in this PR you need to follow this checklist, otherwise feel free to remove.
- [x] Image is marked experimental or stable as appropriate
- [x] The version included is the latest GA version of the software
- [x] The latest tag points to the newest stable version
- [ ] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [x] The image runs as `nonroot` and GID/UID are set to 65532
  - [ ] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
  - [ ] See above for exceptions to nonroot rule
- [x] ENTRYPOINT
  - [x] For applications/servers/utilities call main program with no arguments e.g. [redis-server]
  - [ ] For base images leave empty
  - [ ] For dev variants set to entrypoint script that falls back to system
- [x] CMD:
  - [x] For server applications give arguments to start in daemon mode (may be empty)
  - [ ] For utilities/tooling bring up help e.g. `–help`
  - [ ] For base images with a shell, call it e.g. [/bin/sh]
- [ ] Consider where and how the image deviates from popular alternatives. Is there a good reason and is it documented?
- [x] Add annotations e.g:
```
annotations:
  "org.opencontainers.image.authors": "Chainguard Team https://www.chainguard.dev/"
  "org.opencontainers.image.url": https://edu.chainguard.dev/chainguard/chainguard-images/reference/busybox/ # use the academy site here
  "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/bazel # use github here
```
- [x] Check if environment variables are needed e.g. to set data locations
- [] Ensure the image responds to SIGTERM
  - [ ] `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`
- [ ] Documentation. Let's make this excellent. Include usage example.
- [ ] Error logs write to stderr and normal logs to stdout. DO NOT write to file.
- [x] Include tests, at the very least a basic smoke test.

### CODE REVIEW

File: images/vertical-pod-autoscaler-admission-controller/configs/latest.apko.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+cmd: "--v=4 --stderrthreshold=info"
+    
+archs:
+  - x86_64
+  - aarch64
+
+annotations:
+  "org.opencontainers.image.authors": "Chainguard Team https://www.chainguard.dev/"
+  "org.opencontainers.image.url": https://edu.chainguard.dev/chainguard/chainguard-images/reference/vertical-pod-autoscaler/
+  "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/vertical-pod-autoscaler
```
Comment (jdolitsky) [inline images/vertical-pod-autoscaler-admission-controller/configs/latest.apko.yaml:33]:
```suggestion
  "org.opencontainers.image.url": https://edu.chainguard.dev/chainguard/chainguard-images/reference/vertical-pod-autoscaler-admission-controller/
  "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/vertical-pod-autoscaler-admission-controller
```

File: images/vertical-pod-autoscaler-recommender/image.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+status: experimental
+versions:
+  - apko:
+      config: configs/latest.apko.yaml
+      extractTagsFrom:
+        package: vertical-pod-autoscaler-updater
```
Comment (jdolitsky) [inline images/vertical-pod-autoscaler-recommender/image.yaml:6]:
```suggestion
        package: vertical-pod-autoscaler
```

File: images/vertical-pod-autoscaler-updater/image.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+status: experimental
+versions:
+  - apko:
+      config: configs/latest.apko.yaml
+      extractTagsFrom:
+        package: vertical-pod-autoscaler-updater
```
Comment (jdolitsky) [inline images/vertical-pod-autoscaler-updater/image.yaml:6]:
```suggestion
        package: vertical-pod-autoscaler
```

File: images/vertical-pod-autoscaler-admission-controller/image.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+status: experimental
+versions:
+  - apko:
+      config: configs/latest.apko.yaml
+      extractTagsFrom:
+        package: vertical-pod-autoscaler-admission-controller
```
Comment (jdolitsky) [inline images/vertical-pod-autoscaler-admission-controller/image.yaml:6]:
```suggestion
        package: vertical-pod-autoscaler
```

**Review by jdolitsky — COMMENTED** [review]
are these 3 images all the same but with different entrypoints?

**Comment by developer-guy** [comment]
> are these 3 images all the same but with different entrypoints?

Yes, they are ☝️

File: images/vertical-pod-autoscaler-admission-controller/configs/latest.apko.yaml (hunk 1 above)
Comment (developer-guy) [reply images/vertical-pod-autoscaler-admission-controller/configs/latest.apko.yaml]: thank you!

File: images/vertical-pod-autoscaler-updater/image.yaml (hunk 3 above)
Comment (developer-guy) [reply images/vertical-pod-autoscaler-updater/image.yaml]: thank you!

File: images/vertical-pod-autoscaler-admission-controller/image.yaml (hunk 4 above)
Comment (developer-guy) [reply images/vertical-pod-autoscaler-admission-controller/image.yaml]: thank you!

File: images/vertical-pod-autoscaler-recommender/image.yaml (hunk 2 above)
Comment (developer-guy) [reply images/vertical-pod-autoscaler-recommender/image.yaml]: thank you!

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**